core/src/main/java/org/openstack4j/model/network/ext/ListenerV2.java

```diff
@@ -0,0 +1,71 @@
+package org.openstack4j.model.network.ext;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.openstack4j.common.Buildable;
+import org.openstack4j.model.ModelEntity;
+import org.openstack4j.model.network.ext.builder.ListenerV2Builder;
+import org.openstack4j.openstack.networking.domain.ext.ListItem;
+import org.openstack4j.openstack.networking.domain.ext.NeutronListenerV2;
+
+import java.util.List;
+
+/**
+ * A listener for v2 loadbalancer
+ * @author emjburns
+ */
+@JsonDeserialize(as = NeutronListenerV2.class)
+public interface ListenerV2 extends ModelEntity, Buildable<ListenerV2Builder> {
```

Comment:
It's used for post methods, to create a listener. Update is for updating the listener, it has fewer fields. So it is needed.